zkevm-circuits/src/table.rs

```diff
@@ -2064,7 +2088,12 @@ impl CopyTable {
                 let tag_chip = BinaryNumberChip::construct(self.tag);
                 let copy_table_columns = <CopyTable as LookupTable<F>>::advice_columns(self);
                 for copy_event in block.copy_events.iter() {
-                    for (tag, row, _) in Self::assignments(copy_event, *challenges) {
+                    #[cfg(feature = "dual_bytecode")]
+                    let copy_rows = Self::assignments(copy_event, *challenges, &block.bytecode_map);
+                    #[cfg(not(feature = "dual_bytecode"))]
+                    let copy_rows = Self::assignments(copy_event, *challenges);
```

Comment:
feature flag should be used as less as possible to make code easy to read.  For example, here,  even for `#[cfg(not(feature = "dual_bytecode"))]`, we can still use `Self::assignments(copy_event, *challenges, &block.bytecode_map);`,   but for this case inside the `assignments`, we never access `&block.bytecode_map`